code/modules/hydroponics/gene_strains.dm

```diff
@@ -330,6 +330,10 @@ ABSTRACT_TYPE(/datum/plant_gene_strain)
 		growth_tick.growtime_bonus += 0.1
 		growth_tick.harvtime_bonus += 0.1
 
+/datum/plant_gene_strain/rooted
+	name = "Rooted"
+	desc = "A very rare mutation which allows typically single harvest plants to continue living after a harvest."
```

Comment:
The main concern is really that this just makes them last infinitely long compared to before

Reply:
That would be a lot more amendable imo